xapian-applications/omega/omindexcheck.cc

```diff
@@ -80,6 +80,44 @@ index_test()
     tests.insert({"img/scan-page.png", {"Zannual", "Zfed", "Zreturn",
 		  "Zwhile"}});
 #endif
+#if defined HAVE_LIBARCHIVE
+    // blank file
+    tests.insert({"odf/blank.odt", {}});
+    // corrupted file (ODP)
+    // tests.insert({"corrupt_file.odp", {"ZSnatur"}});
+
+    // ODF
+    tests.insert({"odf/test.odt", {"Zедой", "Z喬伊不分享食物"}});
+    tests.insert({"odf/libarchive_text.odt", {"Stesttitle", "Atestauthor",
+		  "Zsampl", "Zhead", "Ztext", "Zhello",
+		  "Zworld"}});
+    tests.insert({"odf/libarchive_text_template.ott", {"Zjane", "Zdoe",
+		  "Zstructur"}});
```

Comment:
This free-form wrapping makes things harder to read.  I'd wrap between the items instead (and then wrap the list as well but only if it's still too long), e.g.:

```
    tests.insert({"odf/libarchive_text_template.ott",
                  {"Zjane", "Zdoe", "Zstructur"}});
    tests.insert({"odf/libarchive_presentation.odp",
                  {"Zfascin", "Zfact", "Zpustak", "Zmahal", "Zmillion", "Zpeopl",
                   "Zbirthday", "501"}});
```

Reply:
I have changed this.